site/data/dynamo.ts

```diff
@@ -225,7 +248,9 @@ export const upsertDisruptionInfo = async (disruptionInfo: DisruptionInfo, id: s
     logger.info(`Updating draft disruption (${disruptionInfo.disruptionId}) in DynamoDB table...`);
     const currentDisruption = await getDisruptionById(disruptionInfo.disruptionId, id);
     const isEditing =
-        currentDisruption?.publishStatus === "PUBLISHED" || currentDisruption?.publishStatus === "EDITING";
+        currentDisruption?.publishStatus === "PUBLISHED" ||
```

Comment:
I wonder if we should have a publish status enum, as these values seem like good candidates for an enum.

Reply:
done